rasa/constants.py

```diff
@@ -32,7 +32,7 @@
 CONFIG_MANDATORY_KEYS_NLU = ["language", "pipeline"]
 CONFIG_MANDATORY_KEYS = CONFIG_MANDATORY_KEYS_CORE + CONFIG_MANDATORY_KEYS_NLU
 
-MINIMUM_COMPATIBLE_VERSION = "1.0.0rc1"
+MINIMUM_COMPATIBLE_VERSION = "1.3.0"
```

Comment:
```suggestion
MINIMUM_COMPATIBLE_VERSION = "1.3.0a1"
```

Reply:
should I change `version.py` to `1.3.0a1` as well then?

Reply:
already did that on master

Reply:
👍